icekit/project/settings/_base.py

```diff
@@ -414,18 +414,18 @@
 # given model/field, without generating additional unecessary thumbnails. This
 # is essential when using a remote storage backend.
 THUMBNAIL_ALIASES = {
-#     'app[.model][.field]': {
-#         'name-WxH': {
-#             'size': (W, H),
-#             ...,
-#         },
-#     },
+    # 'app[.model][.field]': {
+    #   'name-WxH': { 'size': (W, H), },
+    # },
     '': {
+        'author_portrait': {
+            'size': (360, 640),
+        },
```

Comment:
We should include the WxH in the alias name, especially when it's not namespaced to a particular app or model. We don't know that every `author_portrait` on every site or app will need to be 360x640.